src/fromager/overrides.py

```diff
@@ -50,7 +50,11 @@ def find_and_invoke(
     fn = find_override_method(distname, method)
     if not fn:
         fn = default_fn
-    return invoke(fn, **kwargs)
+
+    logger.debug(f"The arguments for override methods are: {kwargs}")
```

Comment:
I don't think we want to log kwargs here cause not all of them might be used. Take a look at the `invoke` function

Reply:
hmm, okay do we want to log them in `invoke` after ` kwargs.pop(arg_name)` outside for loop?
Or there is no need at all to log kwargs anywhere?

Reply:
I think @tiran wanted the arguments to be logged (#409). So probably log the ones we actually use outside of the if statement

Reply:
okay, will do